src/point.h

```diff
@@ -349,6 +349,19 @@ static constexpr std::array<point, 4> four_adjacent_offsets{{
         point_north, point_east, point_south, point_west
     }};
 
+static constexpr std::array<point, 4> neighborhood{ {
+        point_south, point_east, point_west, point_north
+    }};
+
+static constexpr std::array<point, 4> offsets = {{
```

Comment:
This contains the same values as `four_adjacent_offsets` and the name does not give an indication why one could not simply use the other array and what exactly the difference is. Is this array even needed?

In fact, even `neighborhood` contains the same thing. How is any reader supposed to know what the difference is (the order of the contained points)?

Reply:
While container contents are the same, the order of elements is different (and that matters in some contexts). I am trying to get rid of duplicate code and moving all similar collections to one place is a first step. Ultimately I want to use a single collection.